File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/parking/GetParkingFeeForFleetResponse.java
package com.lattis.ellipse.data.network.model.response.parking;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;

/**
 * Created by ssd3 on 8/1/17.
 */

public class GetParkingFeeForFleetResponse extends AbstractApiResponse {

    @SerializedName("payload")
    GetParkingFeeForFleetDataResponse getParkingFeeForFleetDataResponse;

    public GetParkingFeeForFleetDataResponse GetParkingFeeForFleetData() {
        return getParkingFeeForFleetDataResponse;
    }

}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/parking/GetParkingFeeForFleetResponse.kt
package com.lattis.data.entity.response.parking

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse

class GetParkingFeeForFleetResponse : AbstractApiResponse() {
    @SerializedName("payload")
    var getParkingFeeForFleetDataResponse: GetParkingFeeForFleetDataResponse? = null
    fun getParkingFeeForFleetData(): GetParkingFeeForFleetDataResponse? {
        return getParkingFeeForFleetDataResponse
    }
}